Source/WTF/wtf/PlatformEnable.h

```diff
@@ -209,7 +209,7 @@
 #endif
 
 #if !defined(ENABLE_DESTINATION_COLOR_SPACE_LINEAR_SRGB)
-#define ENABLE_DESTINATION_COLOR_SPACE_LINEAR_SRGB 0
+#define ENABLE_DESTINATION_COLOR_SPACE_LINEAR_SRGB 1
```

Comment:
Maybe this can be enabled in the WPE/GTK cmake files instead of doing it for all platforms?

Reply:
By all platforms you mean everything except Cocoa (for it there is PlatformEnableCocoa.h), right?

Reply:
yes

Reply:
If this is the right fix, we can enable it for all cairo ports. And if both Cococ and cairo ports enable ENABLE_DESTINATION_COLOR_SPACE_LINEAR_SRGB, we can remove the macro entirely.

Reply:
Ok, so I will remove ENABLE_DESTINATION_COLOR_SPACE_LINEAR_SRGB. I need also to check why css3/filters/* LayoutTests fail with my change.

Reply:
I agree with @fujii - we can remove it entirely. The fix looks correct -- for Cairo platforms color-space handling on ImageBuffers is a post-paint operation, Cairo provides sRGB data, if we want linearRGB, we need to convert after the pixels arrived in the ImageBuffer -- and endDrawSourceImage() is the right place for that.

+1 from my side.

Reply:
It will be removed in separate PR to make this change cleaner and increase readability of it.